Src/value.h

```diff
@@ -10,7 +10,7 @@
 #define SEEN_VALUE_H
 
 /* Types of primitives held in Value type - test_loops fails if in other sequence - TODO - verify why */
```

Comment:
Is this TODO still correct? Now the order should not have any impact, BUT the original issue with initialization to the default value might still be present.

Reply:
fixed in next commit